seeds/projects.js

```diff
@@ -0,0 +1,10 @@
+const projects = require('../projects')
+
+exports.seed = function(knex, Promise) {
+  return knex('projects').del()
```

Comment:
Do you think that we need to use DATABASE rather than KNEX here?  The lesson uses KNEX but it might be in our best interest to access our tables in a different way.  We should also think about putting some configuration at the top.